### ISSUE TITLE
Feature-request: format only selected code

### ISSUE THREAD

**Issue by laughedelic** [timonwong/ScalaFormat#4]
Could you implement a feature, that allows to format only the code which is currently selected instead of the whole buffer?

Another feature, based on that, could be "formatting clipboard buffer", so that you can paste already in the write format (or just automatically paste _and_ format).

**Comment by timonwong** [comment]
Yes, it's planned, now I'm working on a python project so I can't give you a schedule, but It will be implemented sooner or later.
Thanks for your feedback!

**Comment by laughedelic** [comment]
Cool, thanks!
